Doc/library/importlib.rst

```diff
@@ -1433,9 +1433,9 @@ an :term:`importer`.
    ``importlib.util.resolve_name('sys', __package__)`` without doing a
    check to see if the **package** argument is needed.
 
-   :exc:`ValueError` is raised if **name** is a relative module name but
+   :exc:`ImportError` is raised if **name** is a relative module name but
    package is a false value (e.g. ``None`` or the empty string).
-   :exc:`ValueError` is also raised a relative name would escape its containing
+   :exc:`ImportError` is also raised a relative name would escape its containing
    package (e.g. requesting ``..bacon`` from within the ``spam`` package).
 
    .. versionadded:: 3.3
```

Comment:
We'll want a `versionchanged` note here, similar to the existing note on `find_spec` below.

Reply:
Thanks, I've added `versionchanged` note.